oidc-ui/src/services/utilService.js

```diff
@@ -0,0 +1,50 @@
+import { validAuthFactors } from "../constants/clientConstants";
```

Comment:
this file can be renamed to walletService

Reply:
resolve in this [f23c62b](https://github.com/mosip/esignet/pull/367/commits/f23c62bca823f172be40fc5ad7a41974c9ebf45b)